mmdet/core/hook/checkloss_hook.py

```diff
@@ -0,0 +1,27 @@
+import torch
+from mmcv.runner.hooks import HOOKS, Hook
+
+
+@HOOKS.register_module()
+class CheckInvalidLossHook(Hook):
+    """Check Invalid Loss Hook.
+
+    This hook will regularly check whether the loss is valid
+    during training.
+
+    Args:
+        interval (int): Checking interval (every k iterations).
+            Default: 10.
+        after_train_iter (bool): Whether check invalid loss in
+            `after_train_iter`. Default: True.
+    """
+
+    def __init__(self, interval=10, after_train_iter=True):
+        self.interval = interval
+        self._after_train_iter = after_train_iter
```

Comment:
this option is unnecessary because it should always be true if users want to use this hook